orthrus:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Wheel Velocity Controllers ---------------------------------------
  red_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: red_coupler_wheel
    pid: {p: 100.0, i: 10.0, d: 0.1}
  green_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: green_coupler_wheel
    pid: {p: 100.0, i: 10.0, d: 0.1}
  blue_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: blue_coupler_wheel
    pid: {p: 100.0, i: 10.0, d: 0.1}

  # Coupler Position Controllers -----------------------------
  red_coupler_position_controller:
    type: effort_controllers/JointPositionController
    joint: red_stabilizer_coupler
    pid: {p: 100.0, i: 0.01, d: 10.0}
  green_coupler_position_controller:
    type: effort_controllers/JointPositionController
    joint: green_stabilizer_coupler
    pid: {p: 100.0, i: 0.01, d: 10.0}
  blue_coupler_position_controller:
    type: effort_controllers/JointPositionController
    joint: blue_stabilizer_coupler
    pid: {p: 100.0, i: 0.01, d: 10.0}
